cfy_manager/components/prometheus/prometheus.py

```diff
@@ -0,0 +1,319 @@
+#########
+# Copyright (c) 2020 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  * See the License for the specific language governing permissions and
+#  * limitations under the License.
+
+
+from os import sep
+from os.path import join
+
+import argh
+
+from ..base_component import BaseComponent
+from ..components_constants import (
+    CONFIG,
+)
+from ..service_names import (
+    PROMETHEUS,
+    NODE_EXPORTER,
+    BLACKBOX_EXPORTER,
+    POSTGRES_EXPORTER,
+    RABBITMQ_EXPORTER,
+)
+from ... import constants
+from ...constants import (
+    CLOUDIFY_USER,
+    CLOUDIFY_GROUP
+)
+from ...logger import get_logger, setup_console_logger
+from ...utils import common, files, service
+
+CONFIG_DIR = join(constants.COMPONENTS_DIR, PROMETHEUS, CONFIG)
+LOG_DIR = join(constants.BASE_LOG_DIR, PROMETHEUS)
+BIN_DIR = join(sep, 'usr', 'local', 'bin')
+SYSTEMD_CONFIG_DIR = join(sep, 'etc', 'systemd', 'system')
+PROMETHEUS_DATA_DIR = join(sep, 'var', 'lib', 'prometheus')
+PROMETHEUS_CONFIG_DIR = join(sep, 'etc', 'prometheus', )
+PROMETHEUS_CONFIG_PATH = join(PROMETHEUS_CONFIG_DIR, 'prometheus.yml')
+PROMETHEUS_VERSION = '2.18.1'
+NODE_EXPORTER_VERSION = '1.0.0'
+BLACKBOX_EXPORTER_VERSION = '0.16.0'
+POSTGRES_EXPORTER_VERSION = '0.8.0'
+RABBITMQ_EXPORTER_VERSION = '1.0.0-RC7'
+
+EXPORTERS = [
+    {
+        'name': BLACKBOX_EXPORTER,
+        # 'version': BLACKBOX_EXPORTER_VERSION,
+        'description': 'Blackbox Exporter',
+        # 'download_url': 'https://github.com/prometheus/{0}/releases/'
+        #                 'download/v{1}/{0}-{1}.linux-amd64.tar.gz'.format(
+        #                     BLACKBOX_EXPORTER, BLACKBOX_EXPORTER_VERSION),
```

Comment:
all of that can be presumably removed now?

Reply:
and the VERSION constants as well

Reply:
done